pkg/virt-api/api.go

```diff
@@ -597,6 +615,14 @@ func (app *virtAPIApp) composeSubresources() {
 						Name:       "virtualmachineinstances/removevolume",
 						Namespaced: true,
 					},
+					{
+						Name:       "virtualmachineinstances/addinterface",
```

Comment:
why the is no `virtualmachines/addinterface`?

Reply:
I join this question... I see we do have other commands associated to virtualmachine, e.g. "virtualmachines/start".

Reply:
I do not see it in this list.